// react-simple-card/lib/components/ImageHeader.js
"use strict";

Object.defineProperty(exports, "__esModule", {
	value: true
});

var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; };

var _react = require("react");

var _react2 = _interopRequireDefault(_react);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

var defaultStyles = {
	position: "relative"
};

var defaultImageStyles = {
	display: "block",
	width: "100%",
	borderRadius: "2px 2px 0 0",
	position: "relative",
	left: "0",
	right: "0",
	top: "0",
	bottom: "0"
};

exports.default = function (_ref) {
	var style = _ref.style,
	    imageSrc = _ref.imageSrc,
	    _ref$alt = _ref.alt,
	    alt = _ref$alt === undefined ? "" : _ref$alt;

	var styles = _extends({}, defaultStyles, style);
	return _react2.default.createElement(
		"div",
		{ className: "card__header--image", style: styles },
		_react2.default.createElement("img", { src: imageSrc, style: defaultImageStyles, alt: alt })
	);
};
